pom.xml

```diff
@@ -141,6 +141,11 @@
         <awaitility.version>4.2.0</awaitility.version>
         <e2e.dependency.skip>true</e2e.dependency.skip>
 
+        <!-- Imap storage dependency package  -->
```

Comment:
Please reference https://github.com/apache/seatunnel/pull/5218/files . 
This jars need move to engine submodule.